packages/react-dom/src/client/ReactDOMFiberInput.js

```diff
@@ -144,6 +144,20 @@ export function initWrapperState(element: Element, props: Object) {
   };
 }
 
+export function updateChecked(
+  element: Element,
+  props: Object,
+  targetType: ?string,
+) {
+  var node = ((element: any): InputWithWrapperState);
+  if (!targetType || targetType === node.type) {
```

Comment:
[In `updateNamedCousins`, we check for radio buttons](https://github.com/facebook/react/blob/master/packages/react-dom/src/client/ReactDOMFiberInput.js#L289-L290) like:

```javascript
function updateNamedCousins(rootNode, props) {
  var name = props.name;
  if (props.type === 'radio' && name != null) {
    // ...
  }
  // ...
}
```

Can you use that check here? That would allow us to remove the `targetType` argument and reading from the DOM.

Reply:
I added the code in updateProperties because updateWrapper also calls updateChecked